external/vcm/vcm/catalog.yaml

```diff
@@ -273,6 +273,22 @@ sources:
       urlpath: "gs://vcm-ml-intermediate/2020-05-coarsen-c384-diagnostics/coarsen_diagnostics/atmos_8xdaily_add_vars_corrected_wind.zarr"
       consolidated: True
 
+  40day_c48_atmos_vwinds:
+    description: Additional v wind 2D dycore variables interpolated to pressure levels from 40-day nudged simulation (May 27 2020 SHiELD),
+      coarsened to C48 resolution. Created by script https://github.com/ai2cm/explore/blob/master/annak/2022-06-17-save-vwind-verif/save_vwind_netcdfs_shield_40d.py
```

Comment:
Nit: Can you fix the fact that in this script currently `CATALOG_VERIF_2D` isn't defined?